### PR TITLE
Add bpillalamarri san diego file

### PR DESCRIPTION
**Pull request by bpillalamarri**

### CODE REVIEW

**Comment by bpillalamarri** [comment]
closes #25

**Comment by pparks01** [comment]
:octocat:

File: bpillalamarri-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# San Diego, CA
+
+##### Places to Eat
+
```
Comment (pparks01) [inline bpillalamarri-sandiego.md]: Wheres a good place to eat?

Comment (bpillalamarri) [reply bpillalamarri-sandiego.md]: 🍨 🍔

**Review by pparks01 — APPROVED** [review]

**Pull request merged**